### ISSUE TITLE
Smallest text in Scratch editor: costume editor dimensions and sound duration

### ISSUE THREAD

**Issue by WorldLanguages** [ScratchAddons/ScratchAddons#6856]
### Why this would be helpful

From feedback:

> is there any way for the SA team to make this text more readable from a distance without having to zoom in
> genuinely infuriating and is one of the worst things about scratch, the dimensions text is so tiny for no good reason

![image](https://github.com/ScratchAddons/ScratchAddons/assets/17484114/38381305-5a4d-4fb4-ab71-b3fc1d0ffd31)

Font size of the dimensions is `0.5rem` and user who sent feedback says it's hard to distinguish the digits 8 and 9.

### My suggestion

I don't know how we should tackle this, but it's unreasonable to zoom-in the whole page each time you want to read the costume editor dimensions.

**Comment by Scythn** [comment]
Perhaps 0.65 could work as a bigger value for it?

**Comment by DNin01** [comment]
The customizable block text style addon could be expanded to other areas, I suppose.

**Comment by Secret-chest** [comment]
0.75

**Comment by Secret-chest** [comment]
> The customizable block text style addon could be expanded to other areas, I suppose.

But it should be separate. Maybe a minimum text size setting? I don't want my whole screen to look like headlines.

**Comment by WorldLanguages** [comment]
> I was also thinking that maybe a hover text can be created so that whenever you hover over a costume sprite it displays the dimensions

**Comment by mxmou** [comment]
Browsers have a minimum font size setting. In Chrome and Edge, it's under Appearance → Customize fonts:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/dd3ca34f-ad29-4896-92e6-c653455e4f9f)

In Firefox, it's under General → Language and Appearance → Fonts → Advanced:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/73f743c5-c738-402f-90f9-664039b8bf67)

If it's set to 10px, the numbers are much more readable:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/9ec2705e-4031-41b6-bab1-e88aed487509)

**Comment by WorldLanguages** [comment]
Thanks mxmou, sent that tip to feedback author

**Comment by DNin01** [comment]
> Browsers have a minimum font size setting.

So this addon/feature isn't necessary then?

**Comment by WorldLanguages** [comment]
> > I was also thinking that maybe a hover text can be created so that whenever you hover over a costume sprite it displays the dimensions

I *guess* we could make this an enabled-by-default easter-egg addon, as realistically no one will want to turn this behavior off. It might generate some minor confusion if people think it's a native Scratch feature, but it's pretty unlikely.
(Note: you would need to hover the costume dimensions, not the costume itself, for the browser to display the HTML title attribute as a readable tooltip)

**Comment by Secret-chest** [comment]
> Browsers have a minimum font size setting.

I don't want to break site designs. I simply need a larger size here.

**Comment by mxmou** [comment]
> > Browsers have a minimum font size setting.
> 
> I don't want to break site designs. I simply need a larger size here.

Most sites don't use such small text. If they do, making it slightly larger probably won't break anything.

**Comment by WorldLanguages** [comment]
> I don't want to break site designs

Yeah, this browser setting changes the minimum font size for *all websites*, which might be inconvinient

**Comment by WorldLanguages** [comment]
Quoting myself:

> I guess we could make this an enabled-by-default (and also easter-egg?) addon, as realistically no one will want to turn this behavior off. It might generate some minor confusion if people think it's a native Scratch feature, but it's pretty unlikely.

Related: "bigger save now button hitbox" (#5189) which IMO more people will notice if we enable it by default without asking.

**Comment by DNin01** [comment]
A really good place to recommend this addon would be on an onboarding page, kind of like in #5209.

**Comment by DNin01** [comment]
It could also be part of a bug-fix collection (#7152).

**Comment by Secret-chest** [comment]
> A really good place to recommend this addon would be on an onboarding page, kind of like in #5209.

We can enable it by default.
